container/test-app/compound/compoundClientAPI.html

```diff
@@ -29,9 +29,16 @@ <h3>
         client-permissions='{"permission": "testPermission"}'
         user-settings='{"language":"it", "date":""}'
         anchor="testAnchorCompound"
-        defer-init="false"
+        defer-init="true"
```

Comment:
```suggestion
        defer-init="false"
```

this should stay false. It seems like this was a leftover mistake from refactoring before. 

Also remove  `compoundContainer.init();` [here](https://github.com/SAP/luigi/pull/3792/files#diff-5240f7fa70d3ab491e4c508543bf0afe51ebd55b587b77b9e66f46e8b403617fR137) . If **defer-init** is **false** , **init()** shouldn't be needed